docs/community/contributing.md

```diff
@@ -54,11 +54,19 @@ To start developing on Django REST framework, first create a Fork from the
 Then clone your fork. The clone command will look like this, with your GitHub
 username instead of YOUR-USERNAME:
 
-    git clone https://github.com/YOUR-USERNAME/Spoon-Knife
+    git clone https://github.com/YOUR-USERNAME/django-rest-framework
 
 See GitHub's [_Fork a Repo_][how-to-fork] Guide for more help.
 
 Changes should broadly follow the [PEP 8][pep-8] style conventions, and we recommend you set up your editor to automatically indicate non-conforming styles.
+You can check conformance each time you commit using the [pre-commit](https://pre-commit.com/) hooks, which we also run on CI.
```

Comment:
Is there an "easier" word than conformance? I'm thinking this may be hard for those where English is not their first language? (Are these docs translated)

IDK maybe I'm being overly pedantic 🤷

Reply:
I changed the wording to "You can check your contributions against these conventions each time you commit using the [pre-commit](https://pre-commit.com/) hooks, which we also run on CI."